src/CashFlows.py

```diff
@@ -1119,15 +1122,15 @@ def __init__(self, **kwargs):
     self._inflation = True
     self._yearlyCashflow = None
 
-  def initParams(self, lifetime, pyomoVar=None):
+  def initParams(self, lifetime, pyomoVar=False):
     """
       Initialize some parameters
       @ In, lifetime, int, the given life time
-      @ In, pyomoVar, pyo.ConcreteModel(), concrete model including all variables
+      @ In, pyomoVar, boolean, 'True' will trigger pyomo flag
```

Comment:
can we clarify this a bit for future developers, something like "If True, indicates that an expression will be constructed instead of a value calculated."